tests/unit_tests/simpll/CMakeLists.txt

```diff
@@ -44,3 +44,12 @@ gtest_discover_tests(runTests)
 add_custom_target(test
   COMMAND ${CMAKE_CTEST_COMMAND} --output-on-failure
   DEPENDS runTests)
+
+message(STATUS "Creating end-2-end tests executable")
+add_custom_target(e2e-test
+    COMMAND python ${CMAKE_BINARY_DIR}/bin/run_e2e_tests.py)
```

Comment:
Technically speaking, tests run by `pytest` are divided into two groups:
- unit tests for Python parts of DiffKemp (located in `tests/unit_tests/*.py`,
- "regression" tests located in `tests/regression`.

I think that only the latter could possibly be called end-to-end tests (probably it's even a better name than "regression tests"). So, if we end up having one executable for executing *all* these tests, I'd prefer calling it `run_pytest_tests.py`.

Alternatively, we could split it into two executables for the two groups.